tap_hubspot/hubspot.py

```diff
@@ -607,7 +607,12 @@ def get_campaign_list(self) -> Iterable:
     def get_campaigns(self):
         for campaign, _ in self.get_campaign_list():
             campaign_id = campaign["id"]
-            resp = self.do("GET", f"/email/public/v1/campaigns/{campaign_id}")
+            try:
+                resp = self.do("GET", f"/email/public/v1/campaigns/{campaign_id}")
+            except requests.HTTPError as e:
```

Comment:
```suggestion
            except requests.HTTPError as e:
                if e.response.status_code == 404:
                    LOGGER.warn(f"campaign{campaign_id} doesn't exist anymore, skipping")
                    continue
                raise
```

Reply:
we only want to skip not found 404 status code. if it is other errors, we need to raise the exception.